ios/KMMICTests/Sources/Dummy/AuthToken+Dummy.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  AuthToken+Dummy.swift
+//  KMMIC
+//
+//  Created by MarkG on 20/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import Shared
+
+@testable import KMMIC
+
+extension AuthToken {
+
+    static let dummy = AuthToken(
```

Comment:
You might need to use `AutoDummyable`: https://github.com/nimblehq/swift-auto-dummy-ios-initiative

Reply:
@suho I tried, but it doesn't work with the model from KMM 🙏

Reply:
@suho FYI, after debugging the Sourcery is unable to get the `init` function from a model in Kotlin